modules/ROOT/pages/to-share-api-asset-to-portal.adoc

```diff
@@ -35,13 +35,18 @@ If no action occurs, refresh your browser, log in again, and add the asset again
 
 Follow these steps to remove an asset from a portal and make the asset private.
 
-Removing an asset from a portal does not delete the asset. It is still available internally.
+Removing an asset from a portal does not delete the asset. The asset is still available internally.
 
-. Select the asset in Exchange and select *Share* -> *Add to the Public Portal*.
-. Uncheck the version field checkbox.
+. Select the asset in Exchange and click *Share*.
+. Select *Public - Publish in the on the public portal*.
+. Unselect the version checkbox.
```

Comment:
```suggestion
. Deselect the version checkbox.
```

Reply:
This is in the style guide, which I think is really weird. I usually write "Clear the checkbox" 

who knows... 🤷

Reply:
I agree with you. I like "clear the checkbox".